demonstrations/ahs_aquila.py

```diff
@@ -0,0 +1,821 @@
+computations. In most digital quantum architectures, the native gates of the computer are, at the
+implementation level, electromagnetic control pulses that have been finely tuned to perform a
+particular logical gate.
+
+This alternative approach requires a different type of control than what you might be used to in
+PennyLane, where circuits are generally defined in terms of a series of gates. Specifically, pulse control
+is implemented via the functionality provided in the Pennylane :mod:`~pennylane.pulse` module. For
+more information on pulse programming in PennyLane, see the
+`PennyLane docs <https://docs.pennylane.ai/en/stable/code/qml_pulse.html>`__, or check out the demo
+about
+`running a ctrl-VQE algorithm with pulse control <https://pennylane.ai/qml/demos/tutorial_pulse_programming101.html>`__.
+
+
+
+The QuEra Aquila device
+-----------------------
+
+The Aquila QPU works with programmable arrays of up to 256 Rubidium-87 atoms (Rb-87), trapped in vacuum by tightly
+focused laser beams. These atoms can be arranged in (almost)
+`arbitrary user-specified geometries <https://pennylane.ai/qml/demos/tutorial_pasqal.html>`_ to determine
+inter-qubit interactions. On the Aquila device, it is possible to specify 1D and 2D atom arrangements. Different
+energy levels of these atoms are used to encode qubits.
+
+A primary application of pulse control in Rydberg atom systems like Aquila is the implementation of analog
+Hamiltonian simulation. This is a technique that aims to investigate the behaviour of some
+system of interest using a programmable, controllable device that emulates the target
+system’s behaviour. For example, Rydberg atom systems have been used to probe the behaviour
+of quantum spin liquids [#Semeghini]_ and antiferromagnetic Ising
+models [#Lienhard]_.
+
+Constructing a pulse program to run on the Aquila hardware is done in two steps, each of which allows us to modify
+different parts of the system Hamiltonian:
+
+1. Define atom positions, which determines qubit connectivity
+2. Specify the quantum evolution via the drive parameters
+
+Let's start with the atom positions and the resulting Hamiltonian term describing inter-qubit interactions.
+
+
+Interaction term and atom arrangement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this treatment of the Hamiltonian, we will assume that we are operating such that we only allow
+access to two states; the low and high energy states are referred to as the ground (:math:`\ket{g}`)
+and Rydberg (:math:`\ket{r}`) states respectively.
+
+Qubit interaction in a Rydberg atom system is mediated by a mechanism called Rydberg blockade, which arises
+due to van der Waals forces between the atoms. This is described by the interaction Hamiltonian:
+
+.. math:: \hat{H}_{int} = \sum_{j=1}^{N-1}\sum_{k=j+1}^{N} V_{jk}\hat{n}_j\hat{n}_k = \sum_{j=1}^{N-1}\sum_{k=j+1}^{N} \frac{C_6}{R^6_{jk}}\hat{n}_j\hat{n}_k
```

Comment:
```suggestion
.. math:: \hat{H}_{int} = \sum_{j=1}^{N-1}\sum_{k=j+1}^{N} \frac{C_6}{R^6_{jk}}\hat{n}_j\hat{n}_k
```
Minor, but do we need the middle part?